api/services/reservationService.js

```diff
@@ -0,0 +1,41 @@
+const reservationDao = require("../models/reservationDao");
+
+const getOptions = async (movies, cinema_names, hall_types, dates) => {
+  const listMovieOptions = await reservationDao.listMovieOptions(
+    movies,
+    cinema_names,
+    hall_types,
+    dates
+  );
+  const listRegionOptions = await reservationDao.listRegionOptions(
+    movies,
+    cinema_names,
+    hall_types,
+    dates
+  );
+  const listHallTypeOptions = await reservationDao.listHallTypeOptions(
+    movies,
+    cinema_names,
+    hall_types,
+    dates
+  );
+  const listDateOptions = await reservationDao.listDateOptions(
+    movies,
+    cinema_names,
+    hall_types,
+    dates
+  );
+
+  const getOptions = [
+    listMovieOptions,
+    listRegionOptions,
+    listHallTypeOptions,
+    listDateOptions,
+  ];
+
+  return getOptions;
```

Comment:
그냥 배열로서 반환하게 되면 프론트엔드에서는 api에서 return해주는 Response의 바디값에서 인덱스에 따라 활용할 수 밖에 없을 것 같은데, 이보다 객체로서 key value를 명시하여 보내주면 어떨까요? 배열의 몇번쨰 요소가 listMovieOptions라는 걸 알려주기보다 객체로서 알려주면 보다 서로 원활 할 것 같습니다.
```suggestion
  return {
    listMovieOptions,
    listRegionOptions,
    listHallTypeOptions,
    listDateOptions,
  };
```